source/MaterialXCore/Node.cpp

```diff
@@ -657,34 +657,33 @@ void Node::addInputsFromNodeDef()
     }
 }
 
-void NodeGraph::addInterfaceName(const string& inputPath, const string& interfaceName)
+InputPtr NodeGraph::addInterfaceName(const string& inputPath, const string& interfaceName)
```

Comment:
This fixes to support adding interfaces on nodegraphs or nodedefs for nodegraphs properly.